packages/services/cdn-worker/src/handler.ts

```diff
@@ -167,6 +177,17 @@ function createArtifactTypesHandlers(analytics: Analytics) {
 const VALID_ARTIFACT_TYPES = artifactTypes;
 const AUTH_HEADER_NAME = 'x-hive-cdn-key';
 
+const legacyToModernArtifactTypeMap: Record<
+  ArtifactType,
+  Extract<ModernArtifactsType, 'sdl' | 'services' | 'metadata' | 'supergraph'>
+> = {
+  schema: 'services',
```

Comment:
Artifacts published to KV as `schema` were either `[{sdl: ""}, {sdl: ""}]` or `{sdl: ""}` but I checked and everyone who requested `/schema` got an array anyway, so it's not a breaking change.